src/store/staking/types.ts

```diff
@@ -0,0 +1,76 @@
+import { DomainRequestAndContents, NftStatusCard } from 'lib/types';
+import {
+	SET_STAKING_REQUESTING_REQUEST,
+	SET_STAKING_REQUESTED_REQUEST,
+	SET_STAKING_APPROVING_REQUEST,
+	SET_STAKING_APPROVED_REQUEST,
+	SET_STAKING_FULFILLING_REQUEST,
+	SET_STAKING_FULFILLED_REQUEST,
+} from './actionTypes';
+
+/**
+ * Staking state definition
+ */
+export type StakingState = {
+	requesting: NftStatusCard[];
+	requested: NftStatusCard[];
+	approving: DomainRequestAndContents[];
+	approved: DomainRequestAndContents[];
+	fulfilling: DomainRequestAndContents[];
+	fulfilled: DomainRequestAndContents[];
+};
+
+/**
```

Comment:
I could be wrong but the comment and type names below look like they have slight typos? is ```stacking``` supposed to be ```staking```?

Reply:
Good call - This is typo